### PR TITLE
Sub-task 1 of array implementations. 

### PR DESCRIPTION
**Pull request by shivhek25**
Solves issue #28 (Sub-task 1)

### CODE REVIEW

**Comment by shivhek25** [comment]
@sahilbansal17 Kindly review.

File: Algorithms/Searching/binary_search.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	cin >> x;
+	cout << "Element to be searched : " << x << endl;
+	binary_search(arr,n,x);
+	return 0;
+}
+
+
+
+//Time Complexity -
+//Worst Case : O(n).
```
Comment (sahilbansal17) [inline Algorithms/Searching/binary_search.cpp]: Is this the complexity for **binary_search** ? Please elaborate.

File: Algorithms/Searching/linear_search.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	cin >> x;
+	cout << "Element to be searched : " << x << endl;
+	linear_Search(arr,n,x);
+	return 0;
+}
+
+
+
+//Time Complexity -
+//Worst Case : O(n).
```
Comment (sahilbansal17) [inline Algorithms/Searching/linear_search.cpp]: Please write one line explanations.

File: Algorithms/Searching/binary_search.cpp (hunk 1 above)
Comment (sahilbansal17) [reply Algorithms/Searching/binary_search.cpp]: Also, follow the naming conventions for functions.

File: Data Structures/Array/deletion.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	cout << "Number of elements : " << n << endl;
+	long long int arr[n];
+	for(long long int i=0; i<n ;i++)
+		cin >> arr[i];
+	cout << "Entered elements : ";
+	printArray(arr,n);
+	long long int x;
+	cin >> x;
+	cout << "Position from which element is to be deleted : " << x << endl;
+	for(long long int i = (x-1); i < (n-1); i++)
```
Comment (sahilbansal17) [inline Data Structures/Array/deletion.cpp]: Write an explanation how are you deleting the elements!

File: Algorithms/Searching/binary_search.cpp (hunk 1 above)
Comment (shivhek25) [reply Algorithms/Searching/binary_search.cpp]: No no, so sorry, I forgot to change the time complexity for binary search. Basically, what I did was copy my linear search code to maintain the template but I forgot to change the time complexity. Will update ASAP. 
Okay.

File: Algorithms/Searching/linear_search.cpp (hunk 2 above)
Comment (shivhek25) [reply Algorithms/Searching/linear_search.cpp]: Sure.

**Comment by shivhek25** [comment]
@sahilbansal17 Kindly review.

**Comment by shivhek25** [comment]
@ramitsawhney27  @aditya81070  Kindly review.

**Comment by sahilbansal17** [comment]
@shivhek25 Kindly fix the indentation issues and maintain the [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards). Update the file as requested by @arnav-mandal1234 .

**Comment by shivhek25** [comment]
@arnav-mandal1234 @sahilbansal17 kindly check

**Pull request merged**